mesh/mesh.go

```diff
@@ -316,7 +317,7 @@ func (msh *Mesh) processValidityUpdates(ctx context.Context, logger log.Log, upd
 					if err := blocks.SetValid(dbtx, bid); err != nil {
 						return err
 					}
-				} else if err := blocks.SetInvalid(dbtx, bid); err != nil {
+				} else if err := blocks.SetInvalid(dbtx, bid); err != nil && !errors.Is(err, sql.ErrNotFound) {
```

Comment:
if we ignore sql.ErrNotFound here, when is the next chance that the block will be set valid in code?
supposedly mesh only gets validity updates from tortoise once when layer is verified

Reply:
i refactored this part